<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleHttpServerClass</class>
 <widget class="QMainWindow" name="SimpleHttpServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleHttpServer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SimpleHttpServer.qrc">
    <normaloff>:/SimpleHttpServer/res/server.png</normaloff>:/SimpleHttpServer/res/server.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	font-family: &quot;Segoe UI&quot;;
	color:#fff;
	background-color:#2D2D2D;
	padding: 2px;
}
QLabel
{
	color:#fff;
	background-color:#2D2D2D;
	padding: 2px;
	border: 0;
}

QPushButton, QTextBrowser, QLineEdit{
	border: 1px solid #EEE;
}

QPushButton:hover {
    background-color: #555;
}

QPushButton:pressed {
    background-color: #5E5E5E;
    color: yellow;
}
QPushButton:disabled {
    background-color: #ccc;
}


QScrollBar:vertical
{
    width:8px;
    background:rgba(200,200,200,8%);
    margin:0px,0px,0px,0px;
    padding-top:9px;
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(255,255,255,8%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(255,255,255,30%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/SimpleHttpServer/res/down.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/SimpleHttpServer/res/up.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/SimpleHttpServer/res/down_d.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/SimpleHttpServer/res/up_d.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(,0,0,10%);
    border-radius:4px;
}
QToolTip {
    background-color: #000000;
    border: 2px solid #333333;
    color: yellow;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ipLineEdit">
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="portLineEdit">
           <property name="text">
            <string>8080</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SupperLineEdit" name="pathLineEdit">
           <property name="toolTip">
            <string>设置服务器跟目录【不支持中文】</string>
           </property>
           <property name="text">
            <string>.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="startPushButton">
           <property name="toolTip">
            <string>开启服务器</string>
           </property>
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>关闭服务器</string>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="SupperLabel" name="serverLabel">
           <property name="toolTip">
            <string>L:用浏览器打开 R:进入作者官网</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="SimpleHttpServer.qrc">:/SimpleHttpServer/res/server.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="logTextBrowser">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>日志</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SupperLineEdit</class>
   <extends>QLineEdit</extends>
   <header>SupperLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SupperLabel</class>
   <extends>QLabel</extends>
   <header>SupperLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SimpleHttpServer.qrc"/>
 </resources>
 <connections/>
</ui>
